Migrate counter and gauge docs to new format (#2007)
timescale/docs: api/_hyperfunctions/gauge_agg/gauge_agg.md

---
api_name: gauge_agg()
excerpt: Aggregate gauge data into an intermediate form for further analysis
topics: [hyperfunctions]
api:
  license: community
  type: function
  toolkit: true
  experimental: true
  version:
    experimental: 1.6.0
hyperfunction:
  family: counters and gauges
  type: aggregate
  aggregates:
    - gauge_agg()
api_details:
  summary: |
    This is the first step for performing any aggregate calculations
    on gauge data. Use `gauge_agg` to create an intermediate aggregate
    from your data. This intermediate form can then be used
    by one or more accessors in this group to compute final results. Optionally,

>     multiple such intermediate aggregate objects can be combined using

Reviewer: ```suggestion
    you can combine multiple intermediate aggregate objects with
```
